### PR TITLE
Add faasflow in the openfaas community reference

### PR DESCRIPTION
**Pull request by s8sg**
Signed-off-by: Swarvanu Sengupta <swarvanusg@gmail.com>

Add [faasflow](https://github.com/s8sg/faasflow) reference in the openfaas community

## Description
Faasflow is library that allows to build openfaas workflow with ease and deploy it as a function, This changes add faasflow reference at `community.md` file

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -292,6 +292,7 @@ You can also find cool projects or submit your own to the [faas-and-furious orga
 
 | Project name and description                                         | Author     | Site      | Date        |
 |----------------------------------------------------------------------|------------|-----------|-------------|
+| [Faasflow - faas workflow as a function](https://github.com/s8sg/faasflow) | Swarvanu Sengupta | github.com | 11-Nov-2018 |
```
Comment (alexellis) [inline community.md]: Might be worth renaming the repo to `faas-flow` to match the naming convention used with other third-party OpenFaaS integrations.

Comment (s8sg) [reply community.md]: It also acts as a golang package. As I remember faas-flow is not a valid package name

**Comment by alexellis** [comment]
Fair point. What I've seen the Go community tending to do is to go with the better name and then add this kind of update: `import faasflow "https://github.com/s8sg/faas-flow"` you'll see it done in Kubernetes and other projects.

**Comment by s8sg** [comment]
I'm still not sure. `https://github.com/s8sg/faas-flow` would have in package name lib files as
```go
package faas-flow // This is invalid
```  
If possible can you please share some example who defines a go package in similar idioms. I'm not sure how to achieve it

**Pull request merged**